enterprise/ha/src/main/java/org/neo4j/kernel/ha/store/StoreUtil.java

```diff
@@ -84,6 +108,35 @@ public static File getBranchedDataDirectory( File storeDir, long timestamp )
         return new File( getBranchedDataRootDirectory( storeDir ), "" + timestamp );
     }
 
+    private static File[] relevantDbFiles( File storeDir )
+    {
+        if ( !storeDir.exists() )
+        {
+            return new File[0];
+        }
+
+        return storeDir.listFiles( file -> {
```

Comment:
we can extract this filter into a private static field to be more close to his close friend `DEEP_STORE_FILE_FILTER`